### ISSUE TITLE
Navigator tree and each of its linker's node of  the each page have the different implementation in http://api.call-cc.org and http://wiki.call-cc.org/

### ISSUE THREAD

**Issue by APIPLM** [ursetto/chicken-doc#9]
Let us say both http://wiki.call-cc.org/eggref/5/srfi-1 and http://api.call-cc.org/5/doc/srfi-1, for the `introduction` node in the page. The first one of implementation for the node is 
   `<a href="#introduction"> <h3 id="introduction">Introduction</h3></a>`. 
The second of the implementation for that node  is 
 `<h3 id="sec:Introduction"><a href="#sec:Introduction">Introduction</a></h3>` 

But the point is that as I view both of the source of the node in `nxml-mode` in Emacs. The first one can not pass the validation.

**Comment by ursetto** [comment]
If I understand you correctly, wiki.call-cc.org's code does not pass validation.

However, the wiki does not use this project -- chicken-doc or chickadee -- those are the backend of api.call-cc.org, which you say validates correctly.

As this would likely be a bug in the wiki, I would suggest asking on the chicken-users mailing list about it.

**Comment by APIPLM** [comment]
Yes. The important point is about the procedure description and document structure, the node implementation of navigator tree can be different.
one more point is that validation of the node implementation of navigator tree passed as I modify the one file .\emacs\share\emacs\25.3\etc\schema\xhtml-hypertext.rnc.  I found out the  api.call-cc.org page use the like abc:s as NCName in the HTML file, which I have not find a way to fix the validation yet.  I found out that your guys are using the different validation  tool for the HTML editor.
Although it is not a question about the chicken. Once I find out an issue or have a question, I will go the chicken-user mailing list to start a thread.
